code/client/cl_console.c

```diff
@@ -665,6 +667,17 @@ void Con_DrawNotify( void )
 	int		skip;
 	int		currentColorIndex;
 	int		colorIndex;
+	
+	if ( con_notifyXY_enable->integer ) 
+	{
+	con.notifyx = con_notifyX->value * (float)cls.glconfig.vidWidth / 640.0;
+	con.notifyy = con_notifyY->value * (float)cls.glconfig.vidHeight / 480.0;
```

Comment:
I'd prefer the following
```suggestion
	float notifyx = con_notifyx->value;
	float notifyy = con_notifyy->value;
	SCR_AdjustFrom640(&notifyx, &notifyy, NULL, NULL);
	con.notifyx = (int)notifyx;
	con.notifyy = (int)notifyy;
```
@runaos what do you think?

Reply:
Is this even needed then? `	con.notifyx = (int)notifyx;
	con.notifyy = (int)notifyy;` 
Remove notifyx and notifyy from the `console_t` struct and use (int)notifyx directly, maybe?

Reply:
Good proposal